code/src/components/app-table/AppTable.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { render, screen } from "@testing-library/react";
+
+import AppTable from "@/components/app-table/AppTable";
+
+describe("AppTable", () => {
+  const headItems = ["Header 1", "Header 2"];
+  const bodyItems = [
+    { id: 1, value: "Row 1" },
+    { id: 2, value: "Row 2" }
+  ];
+
+  const renderHeadItem = (item: string) => <th key={item}>{item}</th>;
+  const renderBodyItem = (item: { id: number; value: string }) => (
+    <tr key={item.id}>
+      <td>{item.value}</td>
+    </tr>
+  );
```

Comment:
all mock items and functions move out of the `describe` test block

Reply:
Moved